src/reviewer.py

```diff
@@ -0,0 +1,38 @@
+import os
+from github_utils import get_pull_request_diff, post_review_comment
+from llm_utils import analyze_code_changes
+
+class AICodeReviewer:
+    def __init__(self):
+        self.openai_api_key = os.getenv('GEMINI_API_KEY')
+        self.github_token = os.getenv('GIT_TOKEN')
+
+    def review_pull_request(self):
+        try:
+            # Get the PR diff
+            diff_content = get_pull_request_diff()
+            if not diff_content:
+                print("No diff content found")
+                return
+
+            # Analyze changes using LLM
```

Comment:
It's generally a good practice to log the diff content itself (or a truncated version) for debugging purposes, especially if there are issues with the review comments.

```suggestion
print(f"Diff Content: {diff_content[:500]}...") # Log first 500 characters
```